assets/docs/sayakpaul/collections/bit-resnet/1.md

```diff
@@ -36,7 +36,11 @@ Models included in this collection have two variants: (1) off-the-shelf inferenc
 ## References 
```

Comment:
Please double check the references here to be sure they match the model names. It looks like the classification models should say `distill_bit_resnet50x1`, while the feature extractors are `distill_bit_r50x1`. Also note that the classification models have been linked twice while the feature extractors are missing.